src/components/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterOptions } from '../enums/FilterOptions';
+
+interface FooterProps {
+  todos: Todo[];
+  completedTodos: Todo[];
+  filter: FilterOptions;
+  setFilter: React.Dispatch<React.SetStateAction<FilterOptions>>;
+  onCompletedTodosDeleted: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  completedTodos,
+  filter,
+  setFilter,
+  onCompletedTodosDeleted,
+}) => {
+  const atLeastOneIsActive = todos.some(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos.length - completedTodos.length} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <nav className="filter" data-cy="Filter">
+          {Object.values(FilterOptions).map(filterOption => (
+            <a
+              key={filterOption}
+              href={`#/${filterOption.charAt(0).toUpperCase() + filterOption.slice(1)}`}
+              className={classNames('filter__link', {
+                selected: filter === filterOption,
+              })}
+              data-cy={`FilterLink${filterOption.charAt(0).toUpperCase() + filterOption.slice(1)}`}
+              onClick={() => setFilter(filterOption)}
+            >
+              {filterOption.charAt(0).toUpperCase() + filterOption.slice(1)}
+            </a>
+          ))}
+        </nav>
+      </nav>
```

Comment:
Avoid Nested <nav> Elements
You have a nested <nav> tag, which isn't semantically correct. You can remove one of them to improve semantics.